build.gradle.kts

```diff
@@ -1,11 +1,11 @@
 plugins {
-    id("org.jetbrains.intellij") version "1.15.0"
+    id("org.jetbrains.intellij") version "1.16.1"
     java
     kotlin("jvm") version "1.8.10"
```

Comment:
v1.8.10 released Feb 2, 2023.

There's been [several Kotlin releases since](https://kotlinlang.org/docs/releases.html#release-details).  Why not a more recent one?

Reply:
I don't use any kotlin-specific features in the plugin. So here is no need to bump version

Reply:
You might not, but the tests appear to use Kotlin...

I'll leave it to you to decide if this needs upgrading.